lib/src/settings.rs

```diff
@@ -143,6 +143,12 @@ impl UserSettings {
             .unwrap_or(false)
     }
 
+    pub fn unique_prefixes(&self) -> String {
```

Comment:
I am planning to make this return an enum rather than a string, similarly to colors. Haven't quite done it yet.

Reply:
Actually, I am leaning towards leaving it as is. I tried making it a type, and it didn't look like an improvement since it can't really be part of the `lib` crate and there's only a single place it's used.